website/www/site/content/en/documentation/patterns/overview.md

```diff
@@ -54,6 +54,10 @@ Pipeline patterns demonstrate common Beam use cases. Pipeline patterns are based
 **State & timers patterns** - Patterns for using state & timers
 * [Grouping elements for efficient external service calls](/documentation/patterns/grouping-elements-for-efficient-external-service-calls/#grouping-elements-for-efficient-external-service-calls-using-the-`GroupIntoBatches`-transform)
 
+**Cache with a shared object** - Patterns for using a shared object as a cache
+* [Create a cache on a batch pipeline](/documentation/patterns/shared-class/#create-a-cache-on-a-batch-pipeline)
```

Comment:
shall we call it out this only works for Python?

Reply:
Let me update to *Patterns for using a shared object as a cache using the Python SDK* in a separate commit.

Reply:
Resolved by a separate PR.